contracts/swappa/RegistryHelperUniswapV2.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.6.8;
+pragma experimental ABIEncoderV2;
+
+import "../interfaces/uniswap/IUniswapV2Pair.sol";
+import "../interfaces/uniswap/IUniswapV2Factory.sol";
+
+contract RegistryHelperUniswapV2 {
+
+    struct PairState {
+        uint reserve0;
+        uint reserve1;
+    }
+
+    struct Pair {
+        IUniswapV2Pair pair;
+        address token0;
+        address token1;
+        PairState state;
+    }
```

Comment:
initial Pair information for initialization, address of pair, tokens, and the initial state